ui/components/app/confirm/info/row/divider.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
@brianacnguyen @georgewrmarshall I remembered one of you mentioned something about the structure of a component 

folder
  |-component
  |-test
  |-story
  |-index
  
For a small component like this, is a story necessary or not? or is it solely by the discretion of the developer.